traffic_portal/test/integration/specs/CDNs.spec.ts

```diff
@@ -39,7 +39,12 @@ cdns.tests.forEach(async cdnsData =>{
             it('can open CDN page', async () => {
                 await cdnsPage.OpenCDNsPage();
             });
-
+             cdnsData.check.forEach(check => {
+                 it(check.description, async () => {
+                     expect(await cdnsPage.CheckCSV(check)).toBeTruthy();
```

Comment:
`CheckCSV` is a `Promise<boolean>`, so this is not merely `Truthy`, it's `True`.

Reply:
Fix to true.